modules/product/src/Tests/ProductEmptyStoreTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\commerce_product\Tests;
+
+use Drupal\commerce\Tests\CommerceTestBase;
+use Drupal\Core\Url;
+
+/**
+ * Empty Store product page test.
+ *
+ * @group commerce
+ */
+class ProductEmptyStoreTest extends CommerceTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'commerce_store',
+    'commerce_product',
+    'commerce_order',
+    'field_ui',
+    'options',
+    'taxonomy',
```

Comment:
Do we need commerce_order, field_ui, options, or taxonomy? Nitpick to try and keep tests a *little* faster by enabling less modules.